src/routes/request/[id]/+page.svelte

```diff
@@ -1,9 +1,63 @@
 <script lang="ts">
-    import type { PageData } from './$types';
+    import { onMount } from 'svelte';
 
-    export let data: PageData;
+    // import { API_TOKEN } from '$env/static/private';
+    let group = 'user datamanagement';
+    const options = { 'delete-account': 'Delete Account', 'delete-user-data': 'Delete User Data' };
+
+    onMount(() => {
```

Comment:
Is this onMount function needed anymore?

If not needed, remove (and remove import)

Reply:
the onMount function is needed to make sure that there is at least one option selected.